variables.tf

```diff
@@ -1,9 +1,3 @@
-variable "create_saml_provider" {
-  description = "Controls whether to create the SAML Identity Provider"
-  type        = bool
-  default     = true
-}
-
 variable "saml_provider_name" {
   description = "Name of the SAML Identity Provider"
   default     = ""
```

Comment:
For this module, let's also delete the default line to make this a required argument.
```suggestion
```

Reply:
Done ... this task has more nuances than I thought.